syncstorage/storage/sql/__init__.py

```diff
@@ -638,10 +639,6 @@ def _prepare_bso_row(self, session, userid, collectionid, item, data):
         row = {}
         row["userid"] = userid
         row["collection"] = collectionid
-        self._prepare_common_row(session, data, item, row)
-        return row
-
-    def _prepare_common_row(self, session, data, item, row):
```

Comment:
It turns out that the row preparation is not that comment between the two queries after all :-)